aiida/backends/tests/cmdline/commands/test_computer.py

```diff
@@ -0,0 +1,106 @@
+"""Tests for the 'verdi computer' command."""
+import os
+import subprocess as sp
+from click.testing import CliRunner
+
+from aiida.backends.testbase import AiidaTestCase
+from aiida.cmdline.commands.computer import (disable_computer, enable_computer)
+from aiida.common.exceptions import NotExistent
+
+
+# pylint: disable=missing-docstring
```

Comment:
pylint disable does not seem necessary here? Even more so given that these files have not been added to pre-commit yet?